website/singlepages/templates/singlepages/responsible_disclosure.html

```diff
@@ -37,13 +40,39 @@ <h4>{% trans "We promise you to" %}</h4>
                 <li>{% trans "take your reports seriously." %}</li>
                 <li>{% trans "respond to your reports as soon as possible." %}</li>
                 <li>{% trans "solve the issue as soon as possible." %}</li>
-                <li>{% trans "respect your privacy and will treat your reports confidentially." %}</li>
+                <li>{% trans "respect your privacy and we will treat your reports confidentially." %}</li>
```

Comment:
Do you think 'we' should be added here?

Reply:
I think 'will' should be removed

"... privacy and treat your ..."

Reply:
I updated the file.